docs/reference/functions/rounding/significants.html

```diff
@@ -0,0 +1,72 @@
+<!--
+  EVE -  Expressive Vector Engine
+  Copyright 2020 Joel FALCOU
+  Copyright 2020 Jean-Thierry LAPRESTE
+
+  Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+  SPDX-License-Identifier: MIT
+ -->
+<meta charset="utf-8" lang="en">
+                       **Expressive Vector Engine**
+                           `eve::significants`
+<br>
+
+  (insert ../../../crumbs.html here)
+
+Synopsis
+====================================================================================================
+**Required header:** <script type="preformatted">`#include <eve/function/significants.hpp>`</script>
+
+<script type="preformatted">
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ c++
+namespace eve
+{
+  inline constexpr /*unspecified*/ significants = /*unspecified*/;
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+</script>
+Function object representing the significants operation.
+
+Member functions
+====================================================================================================
+<script type="preformatted">
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ c++
+template< floating_value T,integral_value U > auto operator()( T x, U n ) const noexcept;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+</script>
+Performs the  [element-wise](../../../glossary.html#element-wise) significants operation on its parameter.
+
+
+Parameters
+----------------------------------------------------------------------------------------------------
+`x`
+:   [floating value](../../concepts.html#value).
+`n`
+:   [integral value](../../concepts.html#value).
+
+Return value
+----------------------------------------------------------------------------------------------------
+<span class="smallskip"></span>
+Computes  [element-wise](../../../glossary.html#elment-wise) the rounding to n significants digits of `x`.
+With null n the result is a nan.
+
+!!! WARNING
```

Comment:
The part abotu it beign an helper adds no infos. Just states it has accuracy limitation.